end-to-end-test/local/specs/treatment.spec.js

```diff
@@ -82,29 +82,29 @@ describe('treatment feature', function() {
             it('shows all treatments in the treatment select box', () => {
                 openHeatmapMenu();
                 selectReactSelectOption( $('.oncoprint__controls__heatmap_menu'), 'IC50 values of compounds on cellular phenotype readout');
-                var treatments = getReactSelectOptions( $('.oncoprint__controls__heatmap_menu .treatment-selector') );
+                var treatments = getNewReactSelectOptions( $('.oncoprint__controls__heatmap_menu .treatment-selector') );
                 assert.equal(treatments.length, 10);
             });
 
             it('adds treatment to icons when selected in treatment select box', () => {
                 openHeatmapMenu();
                 selectReactSelectOption( $('.oncoprint__controls__heatmap_menu'), 'IC50 values of compounds on cellular phenotype readout');
-                var treatments = getReactSelectOptions( $('.oncoprint__controls__heatmap_menu .treatment-selector') );
+                var treatments = getNewReactSelectOptions( $('.oncoprint__controls__heatmap_menu .treatment-selector') );
                 var treatment = treatments[0];
                 var treatmentName = treatment.getText();
                 treatmentName = treatmentName.replace(/.*\((.*)\).*/, "$1")
-                treatment.click();
+                treatment.$('..').click();
```

Comment:
what does this do?

Reply:
This is to get the parent of treatment element because the treatment element itself couldn't receive the click event.